L/LLVMLibUnwind/LLVMLibUnwind@18.1.7/build_tarballs.jl

```diff
@@ -0,0 +1,7 @@
+version = v"18.1.7"
+
+include("../common.jl")
+
+# Build the tarballs
+build_tarballs(ARGS, configure(version; experimental=true)...;
+               preferred_gcc_version=v"10", preferred_llvm_version=v"17", julia_compat="1.12")
```

Comment:
```suggestion
               preferred_gcc_version=v"10", preferred_llvm_version=v"18", julia_compat="1.12")
```

Try 18 given the missing symbol issues?